### PR TITLE
Alternate feature branch

### PR DESCRIPTION
**Pull request by HeflerDev**
Finished New York Times project

### CODE REVIEW

**Review by pelzolga123 — CHANGES_REQUESTED** [review]
Project requires changes.

**Comment by pelzolga123** [comment]
## Hi Team!  
### Your project looks :+1: amazing!
But there are few things that need to be corrected:
#### General
* [x] check HTML markup with this [validator](https://validator.w3.org/#validate_by_upload). Now it shows errors in your project.
* [x] README.md file is not descriptive.
* [x] Stickler errors should be fixed.
* [x] Id should be unique for each element. (You have same id `first-line` for multiple elements. Better use class instead.)

#### Navigation bar
* [x] The NY Times logo should be a link
#### Bottom part
* [x] It should contain links to other articles with image previews on the leftmost side. (Right now you have `p` tags there, but they need to be links). 
#### Footer
* [x] It should have a fixed position (it sticks to the bottom part of the screen) 
Like on original website:
image

 #### Main content
* [x] 'Theory of inflation' part is missing.
image
* [x] 'Read more' button is not needed, please do the whole page instead.
image

###  Please kindly make this changes and send another code review request.

If you have any questions or need help, you can message me on slack. *Happy coding*

File: index.html [hunk 1]
```diff
@@ -0,0 +1,551 @@
...
+                                <strong>FLUCTUATION</strong>  Astronomers have now detected evidence of these ancient fluctuations in swirls of polarized light in the cosmic background radiation, which is energy left over from the early universe. These are gravitational waves predicted by Einstein.
+                            </p>
+                        </td>
+                        <td>
+                            <p>
+                                <strong>EXPANSION</strong>  Returning to our coffee, imagine a single, central pot expanding faster than light and cooling to an even temperature as it expands. That is something like inflation. And the structure of the universe mirrors the froth and foam of the original pot.
+                            </p>
+                        </td>
+                    </tr>
+                </table>
```
Comment (mosesogwo) [inline index.html]: All elements should be aligned using float.

**Review by mosesogwo — CHANGES_REQUESTED** [review]
## :clap: Well done guys
### Only one more requirement left for you to :airplane: 

You have used a table to align items in the **theory of inflation** section. You are required to use float to align all items in this project. (See project specific section on pathwright).



You're almost there. Keep the good work up.

**Status: Changes required**

**Review by nikasulo — APPROVED** [review]
# Congratulations :tada: you may merge!
You did such a great job implementing the requested changes :smile: please keep the energy moving forward :+1: . See you at the top :rocket:.

**Pull request merged**